SwiftUIProject/SwiftUIProject/StartView.swift

```diff
@@ -95,7 +100,7 @@ struct StartView: View {
     }
 
     private var getStartedButtonView: some View {
-        NavigationLink(destination: DetailView()) {
+        Button {} label: {
```

Comment:
```suggestion
        Button {
        viewModel.isProgressViewShown.toggle()
        iewModel.progressViewActivate()
        } label: {
```
можно перетащить и не использовать гесчур